pkg/porter/publish_test.go

```diff
@@ -84,63 +84,63 @@ func TestPublish_validateTag(t *testing.T) {
 
 func TestPublish_getNewImageNameFromBundleReference(t *testing.T) {
 	t.Run("has registry and org", func(t *testing.T) {
-		newInvImgName, err := getNewImageNameFromBundleReference("localhost:5000/myorg/apache-installer", "example.com/neworg/apache:v0.1.0")
+		newInvImgName, err := getNewImageNameFromBundleReference("localhost:5000/myorg/apache-installer", "example.com/neworg/apache:v0.1.0", true)
```

Comment:
I am wondering if we actually need the `isInvocationImage` flag for this function? Seems like since Porter is always passing in the invocation image name from the bundle, e.g. "example/mybundle-installer:v0.1.1", we can just use that value same as we use the image names of referenced images.

Want to try that out and see if it works?

Reply:
I don't quite understand why it's different for the invocation image, when the referenced images also are going to be in the old location after archiving. I would have thought that they would both have the same problem then. Do you have an example of what the names look like?

Reply:
Yes, for example in this test, https://github.com/getporter/porter/blob/cb68d13594854ed8286c0a38591115ede2147bf0/pkg/porter/publish_test.go#L114. 
Do we want the new image to be placed under the bundle repo or keep it under the old location?
the difference would be `getporter.azurecr.io/newor/whalesayd:497b3e01ed1153e542a19df184607866` or `getporter.azurecr.io/neworg/whalegap:497b3e01ed1153e542a19df184607866`

Reply:
With this feature all images (both referenced and invocation images) should be stored in the same bundle repository.

Reply:
I see. I updated the code to reflect this